src/intents/DSL/actions/blockIntervalDutchAuctionSwapAction.ts

```diff
@@ -65,8 +65,8 @@ function blockIntervalDutchAuctionSwap ({
       functionName: 'swap01',
       params: {
         signer: owner,
-        tokenIn: tokenIn as SegmentParamValue,
-        tokenOut: tokenOut as SegmentParamValue,
+        tokenIn: { address: tokenIn.address } as SegmentParamValue,
```

Comment:
@mikec can you tell me whether it's correct? The thing is tokenIn and tokenOut contain values that are irrelevant for segments.

Reply:
I think this will work because we are not supporting other values for token struct to be set with DSL. The additional struct values were for ERC721 (NFT's) which we're not currently dealing with. If tests for DSL are passing, this should be ok.